api/v2/views/volume.py

```diff
@@ -104,6 +103,9 @@ def perform_create(self, serializer):
             return inactive_provider(pna)
         except VOLUME_EXCEPTIONS as e:
             raise exceptions.ParseError(detail=e.message)
+        except Exception as exc:
+            logger.exception("Error occurred creating a v2 volume")
```

Comment:
Added 'user' *only* because:
1. Its guaranteed to be available && will help track down specific users who can complain.
2. volume alias isn't available here (Because create failed)